packages/main/src/DatePicker.js

```diff
@@ -464,6 +464,11 @@ class DatePicker extends WebComponent {
 	}
 
 	_changeCalendarSelection() {
+		if (this._calendarDate.getYear() < 1) {
+			// 0 is a valid year, but we cannot display it
+			return;
+		}
```

Comment:
discussed, as 0 is valid lets display it.

Reply:
there is a problem in islamic. 1 is the minimum for now